srcpkgs/curl/template

```diff
@@ -43,6 +43,9 @@ pre_configure() {
 	export CFLAGS="${CFLAGS/-D_FORTIFY_SOURCE=2/}"
 	export CFLAGS="${CFLAGS/-I${XBPS_CROSS_BASE}\/usr\/include/}"
 	export CPPFLAGS="-D_FORTIFY_SOURCE=2"
+
+	# reconfigure for patched changes
+	autoreconf -fi
```

Comment:
Just make man and do install manually!

Reply:
or we switch to the cmake build?

Reply:
doing autoreconf felt less invasive to me, as for cmake - I'm not sure what the benefit is when the autotools config is tried and true at this point

Reply:
I merge as is. The resulting package is the very same.